File: service/src/main/java/hu/bsstudio/robonaut/race/skill/timer/DefaultSkillTimerService.java
package hu.bsstudio.robonaut.race.skill.timer;

import hu.bsstudio.robonaut.race.skill.timer.model.SkillTimer;
import lombok.RequiredArgsConstructor;
import reactor.core.publisher.Mono;

@RequiredArgsConstructor
public class DefaultSkillTimerService implements SkillTimerService {

    @Override
    public Mono<SkillTimer> updateTimer(final SkillTimer skillTimer) {
        return Mono.just(skillTimer);
    }
}

File: server/service/src/main/kotlin/hu/bsstudio/robonaut/race/skill/timer/DefaultSkillTimerService.kt
package hu.bsstudio.robonaut.race.skill.timer

import hu.bsstudio.robonaut.race.skill.timer.model.SkillTimer
import reactor.core.publisher.Mono

class DefaultSkillTimerService : SkillTimerService {
  override fun updateTimer(skillTimer: SkillTimer): Mono<SkillTimer> = Mono.just(skillTimer)
}
